e2e/e2e-script.sh

```diff
@@ -123,9 +120,9 @@ clusterInfoEndTime=$(date +%s)
 log "Retrieved cluster info in $((clusterInfoEndTime-clusterInfoStartTime)) seconds"
 
 set +x
-addJsonToFile "apiserver.crt" "$(cat apiserver.crt)"
-addJsonToFile "ca.crt" "$(cat ca.crt)"
-addJsonToFile "client.key" "$(cat client.key)"
+addJsonToFile "apiserverCrt" "$(cat apiserver.crt)"
```

Comment:
changed from file format to camel case because bash doesnt allow exporting variables with special chars.